### DISCUSSION TITLE
Update NEW indicator on Activity tab

### DISCUSSION THREAD

**Discussion by kandarpksk** [chanzuckerberg/napari-hub#987] [Ideas]
It's about 4 months since the Activity tab was released, if I'm remembering correctly. I'm wondering what we should do about the NEW indicator, to avoid reader fatigue/blindness when we use it elsewhere in the future.

Suggested options:
- Decide how long we consider a hub feature as new, so we know when to come back to this!
- Remove the indicator. We can consider only showing it for new users via say, AppCues, if that's possible to track. Maybe "new" in this case could be defined as a user who hasn't seen the Activity tab, so once they've opened it, they no longer see the indicator.
- Change the indicator to say "UPDATED" (or something similar). It's also been a while since we have added the maintenance section, but that's something to highlight in case users missed it!

@richaagarwal @codemonkey800 @codemonkey800 @Janeece

**Comment by Nadalyn-CZI** [comment]
When new products (like laundry detergent) come on the market, they can say "New" on the label for up to 6 months in the U.S.
